andela/mazus-ah-frontend
# #167165024 User Should Be Able To Comment On An Article

### What does this PR do?
User should be able to comment on articles

#### Description of Task to be completed?
Adds a comment input field
Adds cards for fetched comments
Refactored tests to accommodate change

#### How should this be manually tested?
git fetch origin ft-comment-on-articles-167165024
npm run dev 
navigate to the single article page and scroll down 
you should be able to comment on an article

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167165024](https://www.pivotaltracker.com/story/show/167165024)

#### Screenshots (if appropriate)
<img width="849" alt="Screenshot 2019-09-10 at 11 41 29 AM" src="https://user-images.githubusercontent.com/26358047/64607410-5f720400-d3c0-11e9-97f5-19f939c8c9bd.png">
